src/NServiceBus.Persistence.ServiceFabric/SagaPersister/SagaPersister.cs

```diff
@@ -28,13 +24,6 @@ public Task Complete(IContainSagaData sagaData, SynchronizedStorageSession sessi
                 {
```

Comment:
The 2nd part checks if there's a concurrent process that has completed that same saga. Removing it would not change the course of things, but would hide the contention on the saga data.